events/sinks/manager_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sinks
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	kube_api "k8s.io/kubernetes/pkg/api"
+
+	"k8s.io/heapster/events/core"
+	"k8s.io/heapster/events/util"
+)
+
+func TestAllExportsInTime(t *testing.T) {
+	timeout := 3 * time.Second
+
+	sink1 := util.NewDummySink("s1", time.Second)
```

Comment:
Please extract common parts into 'processBatch(sink1sleep, sink2sleep) (elapsed, sink1export, sink2export) or sth like that.